python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Events.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>
+    LEDBasicState_Unknown=0,
+    LEDBasicState_Off=1,
+    LEDBasicState_On=2,
+    LEDBasicState_UnexpectedlyOn=7,
+    LEDBasicState_UnexpectedlyOff=8
+  </Enumeration>
+  <Enumeration>
+    LabjackControllerState_Unknown=0,
+    LabjackControllerState_StandbyOrOn=1,
+    LabjackControllerState_Cooldown=2,
+    LabjackControllerState_Error=3
+  </Enumeration>
+  <SALEvent>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_logevent_labjackConnected</EFDB_Topic>
+    <Description>Is the CSC connected to the labjack controller interfacing with the LEDs?</Description>
+    <item>
+      <EFDB_Name>connected</EFDB_Name>
+      <Description>Connected if true, else disconnected</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_logevent_labjackState</EFDB_Topic>
+    <Description>Labjack interface state</Description>
+    <item>
+      <EFDB_Name>controllerState</EFDB_Name>
+      <Description>Labjack controller state; a LabjackControllerState enum.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_logevent_ledState</EFDB_Topic>
+    <Description>LED status list for all LEDs connected</Description>
+    <item>
+      <EFDB_Name>numericalID</EFDB_Name>
+      <Description>List of LED numerical index correlating with list data</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>10</Count>
+    </item>
+    <item>
+      <EFDB_Name>serialNumber</EFDB_Name>
+      <Description>List of LED serial-numbers correlating with list data</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>10</Count>
+    </item>
+    <item>
+      <EFDB_Name>ledState</EFDB_Name>
+      <Description>LED state; a LEDBasicState enum.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>10</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_logevent_ledOnMinutes</EFDB_Topic>
```

Comment:
Consider renaming this event to something like `ledDutyCycle`. 

When do you imagine this event will be published? Something like, when the user sends an on/off command? I imagine you don't want to be publishing this constantly right?